website/hw/hw7.rst

```diff
@@ -231,65 +250,85 @@ repository:
 That way, we can check that you’ve followed the steps described in this
 task. You do not need to enter anything into Gradescope for this task.
 
-Task 3: Create .travis.yml for HelloApp
-=======================================
+Task 3: Create GitHub Actions for HelloApp
+------------------------------------------
 
 [40 points]
 
-As you may remember from our [previous lab]({{< relref “lab6.md” >}}),
+As you may remember from our `previous homework <hw6.html>`__,
 we have a way to make sure our app passes all the tests every time we
 push.
 
-For this, you should create a ``.travis.yml`` file in your
-``2020-lab7-GITHUB_USERNAME`` repo. With Python, there’s no need to
-build, so you should just define a single job that runs the command
-``pytest`` as its test script (in a Linux environment)
+For this, you should create a ``.github/workflows/test-app.yml`` file in your
+``2021-hw7-GITHUB_USERNAME`` repo. With Python, there’s no need to
+build, so your job should only do the following:
```

Comment:
Because of the initial workflow template given in Homework 6 I defaulted to using `ubuntu-16.04`, which gives a syntax error when trying to run `pytest` (I assume this is due to Python 2.x and 3.x version mismatch). You can find an example here: https://github.com/uchicago-cmsc22000/2021-hw7-firatciftci/runs/2558436007.

Should we tell the students to use `ubuntu-latest`, or do we want them to figure it out on their own?

Reply:
This is not something we expect them to figure out on their own. I've added a note about this.